com.unity.template-hd/Packages/manifest.json

```diff
@@ -1,14 +1,14 @@
 {
   "dependencies": {
     "com.unity.cinemachine": "2.8.0",
-    "com.unity.ide.rider": "3.0.7",
-    "com.unity.ide.visualstudio": "2.0.11",
-    "com.unity.ide.vscode": "1.2.3",
+    "com.unity.collab-proxy": "1.9.0",    
+    "com.unity.feature.development": "1.0.1",
     "com.unity.inputsystem": "1.1.0-pre.5",
     "com.unity.learn.iet-framework": "2.0.0",
     "com.unity.render-pipelines.high-definition": "12.0.0",
```

Comment:
The version in package.json seems to be different, it is likely going to be flagged by Yamato when it runs

Reply:
good catch